src/components/products/ProductInfo.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Heading } from '../Heading';
+import Stars from '../shared/Stars';
+import { Button } from '../shared';
+import { Counter } from '../inputs/Counter';
+import { LinkFacade } from '../../lib';
+
+type ProductInfoProps = {
+  title: string;
+  category: string;
+  rating: number;
+  price: number;
+  description: string;
+};
+
+export function ProductInfo({
+  title,
+  category,
+  rating,
+  price,
+  description,
+}: ProductInfoProps) {
+  const [quantity, setQuantity] = useState(1);
+
+  return (
+    <section className="grid gap-5">
+      <header className="grid gap-2">
+        <Heading title={title} subtitle={category} />
+        <div className="flex items-center justify-start gap-2">
+          <Stars num={rating} />
+          <LinkFacade to={`/products/${category}`}>
+            <span className="text-sm text-primary">125 Reviews</span>
+          </LinkFacade>
+          <span className="px-1 text-sm border-l text-neutral-500 dark:text-neutral-400 border-neutral-700">
+            123 Orders
+          </span>
+        </div>
+      </header>
+      <main className="grid gap-5">
+        <article className="text-sm text-neutral-500 dark:text-neutral-400 ">
+          {description}
+        </article>
+        <h2 className="text-2xl font-bold dark:text-white">${price}</h2>
+        <span className="flex items-center gap-2 text-sm text-neutral-500 dark:text-neutral-400">
+          Stock:{' '}
+          <span className="px-4 py-2 font-bold text-black border rounded-md shadow-sm dark:text-white">
```

Comment:
I think you missed this.
I think you should add: ```dark:text-white``` because the number inside the stock input is practically invisible in light mode,

Reply:
Awesome. Thanks.

Reply:
Awesome. Thanks.